driver/main.c

```diff
@@ -56,9 +56,8 @@ or GPL2.txt for full copies of the license.
 #include "ppm_events.h"
 #include "ppm.h"
 #include "ppm_tp.h"
-#if defined(CONFIG_IA32_EMULATION) && !defined(__NR_ia32_socketcall)
-#include "ppm_compat_unistd_32.h"
-#endif
+
+#define __NR_ia32_socketcall 102
```

Comment:
are we sure it has the same value on all supported archs? (not sure if we support other 32-bit architectures apart from x86, if no please ignore the comment)

Reply:
ia32 is a specific arch (x86-32). I'm not sure the kernel supports any other dual 32/64 bit syscall set in a single arch like x86. I'm pretty sure the compat magic is x86(32,64) specific.

BTW, there's also the x32 arch (64-bit with 32-bit pointers) we should do something about... (https://en.wikipedia.org/wiki/X32_ABI)